pkg/scheduler/objects/node.go

```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/apache/yunikorn-scheduler-interface/lib/go/si"
 )
 
+const (
+	UnknownInstanceType = "unknown"
```

Comment:
Maybe we want unknown nodes to stand out a bit more -- perhaps "OTHER" in uppercase?

Reply:
Sure, will change in next rev.